csrc/python_frontend/python_bindings.cpp

```diff
@@ -418,7 +418,7 @@ void initNvFuserPythonBindings(PyObject* module) {
       .def_static(
           "get",
           &FusionCache::get,
-          py::arg("max_fusions") = int(8192),
```

Comment:
do we need to have these? I thought we'd want to rely on overload if the defaults are going to be the same anyway.

Reply:
sorry, not sure i'm following :-(

are you saying that we should get rid of the python `max_fusions` hardcoded as the same int as in c++, and just have the python read the c++ setting? if so, yes, strong agree.

Reply:
Sounds like you know exactly what I was saying :) 

No worries I'll clean it up later.

Reply:
> Sounds like you know exactly what I was saying :)
😄 
> No worries I'll clean it up later.

😃 thanks!